.rubocop.yml

```diff
@@ -43,6 +43,9 @@ Style/IfInsideElse:
 Style/RedundantReturn:
   Enabled: false
 
+Style/WordArray:
+  MinSize: 5
```

Comment:
PR opened to update Rubocop Style/WordArray documentation: https://github.com/bbatsov/rubocop/pull/3014